hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -198,7 +200,17 @@ public class HoodieTableConfig extends HoodieConfig {
       .key("hoodie.timeline.layout.version")
       .noDefaultValue()
       .withDocumentation("Version of timeline used, by the table.");
-  
+
+  public static final ConfigProperty<String> SUPPLEMENTARY_TABLE_FORMAT_CLASS_NAME = ConfigProperty
+      .key("hoodie.table.supplementary.format.class.name")
```

Comment:
rename: `hoodie.table.format` and it takes a string/enum.. defaults to `native`

lets not add any class names into table property.. lets use enums. or strings to make table configs language agnostic.